fractions/monitor/src/main/java/org/wildfly/swarm/monitor/runtime/InstallMonitorFilter.java

```diff
@@ -47,10 +42,11 @@
 
     @Override
     public void prepareArchive(Archive<?> archive) {
-        JARArchive jarArchive = archive.as(JARArchive.class);
-
-        Asset resource = new ClassAsset(HealthResponseFilter.class);
-        ArchivePath location = new BasicPath(PATH_CLASSES, AssetUtil.getFullPathForClassResource(HealthResponseFilter.class));
-        jarArchive.add(resource, location);
+        try {
```

Comment:
ignore this comment, just realized that monitor doesn't have a "deployment" module that things could be exposed from.

However, is there a method for archives where we can add a GA dependency, or is that only inside a main()?